.ansible-lint

```diff
@@ -4,11 +4,17 @@
 # https://github.com/ansible/ansible-lint/issues/371
 # exclude_paths:
 #   - roles/master_role_example/
+exclude_paths:
+  - '.github/'
+  - 'changelogs/'
 parseable: true
 use_default_rules: true
 # https://github.com/ansible/ansible-lint/issues/808
 #  with verbosity set to 1, its dumping 'unknown file type messages'
 # verbosity: 1
 skip_list:
   - meta-runtime
+warn_list:
+  - jinja[invalid]  # Temporarily adding this due to https://github.com/ansible/ansible-lint/issues/3048
+  - jinja[spacing]  # Would be set to warn by default and the only flagged issues are fairly unfixable
```

Comment:
Why are we not failing on spacing here?

Reply:
It started failing when I added this warn_list, so I assume it was set as a warn rule before. It's only a couple of lines and if you look at them they can't really be fixed anyway

Reply:
I also realise that may not have been on this repo but it made sense to make all the repos skip the same rules